### PR TITLE
Fix deps for missing new chain-test-utils

### PR DESCRIPTION
**Pull request by danielSanchezQ**
Removed unexistent feature-test-api from chain-core deps

### CODE REVIEW

**Comment by danielSanchezQ** [comment]
Depends on [this](https://github.com/input-output-hk/chain-libs/pull/375/files)

**Comment by mzabaluev** [comment]
https://github.com/input-output-hk/chain-libs/pull/375 has been merged to master. Please update chain-deps to master and rebase.

File: jormungandr-lib/Cargo.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ quickcheck = "0.9"
 chain-impl-mockchain = { path = "../chain-deps/chain-impl-mockchain", features = [ "property-test-api" ] }
 chain-addr      = { path = "../chain-deps/chain-addr", features = [ "property-test-api" ] }
 chain-crypto    = { path = "../chain-deps/chain-crypto", features = [ "property-test-api" ] }
-chain-core    = { path = "../chain-deps/chain-core", features = [ "property-test-api" ] }
+chain-core    = { path = "../chain-deps/chain-core"}
```
Comment (mzabaluev) [inline jormungandr-lib/Cargo.toml:33]: The missing space before the closing brace bugs me, here and below :grin: Otherwise looks good.

Comment (danielSanchezQ) [reply jormungandr-lib/Cargo.toml:33]: It bugs me too actually 🤦

**Pull request merged**